CIP-0129/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Binary format
+
+In the header-byte, bits [7;4] indicate the type of gov key being used. The remaining four bits [3;0] are used to define the credential type. There are currently 3 types of credentials defined in the Cardano Conway era, this specification will allow us to define a maximum of 16 different types of keys in the future.
+
+```
+  1 byte     variable length   
+ <------> <-------------------> 
+┌────────┬─────────────────────┐
+│ header │        key      │
+└────────┴─────────────────────┘
+    🔎                          
+    ╎          7 6 5 4 3 2 1 0  
+    ╎         ┌─┬─┬─┬─┬─┬─┬─┬─┐ 
+    ╰╌╌╌╌╌╌╌╌ |t│t│t│t│c│c│c│c│ 
+              └─┴─┴─┴─┴─┴─┴─┴─┘ 
+```
+
+#### Key Type
+There are currently 3 types of governance keys, and it is defined using the first half (bits [7;4]) of the header. The different types are summarized below,
+
+Key Type (`t t t t . . . .`)      | Key
+---                               | ---
+`0000....`                        | CC Hot 
+`0001....`                        | CC Cold
+`0010....`                        | DRep
+
+
+#### Credential Type
+
+The second half of the header (bits [3;0]) refers to the credential type which can have the values and types as summarized in the table below,
+
+We *reserve* the values 0 and 1, as these are used in the Cardano Address Network tag. By not using these values we will make the gov identifies not become accidentally compatible with a valid address.
+
+Credential Type (`. . . . c c c c`)   | Semantic
+---                                   | ---
+`....0010`                            | Key Hash 
+`....0011`                            | Script Hash
+
+### Gov Action
+
+Cardano Conway era introduces proposal procedures to submit gov actions. Gov actions are vote on chain by different kinds of keys, and as such it is necessary to be able to share gov actions across communication channels. The gov action is currently defined as combination of a transaction ID it was submitted in and an index within the transaction.
+
+We define a byte format to combine the tx ID and the index to form a single valid byte string, as such it can be converted into a hex format and have its own bech32 prefix.
+
+Transaction ID is always a 32 byte length, hence we can append the index bytes to the tx id, please see below
+
+**Example 1**
+TX ID in Hex - `0000000000000000000000000000000000000000000000000000000000000000`
+Gov Action index in HEX - `11` (number 17)
+gov action ID - `000000000000000000000000000000000000000000000000000000000000000011`
+bech32 - `gov_action1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqpzklpgpf`
+
+**Example 2**
+TX ID in Hex - `1111111111111111111111111111111111111111111111111111111111111111`
+Gov Action index in HEX - `00` (number 0) 
+gov action ID - `111111111111111111111111111111111111111111111111111111111111111100`
```

Comment:
Gov action id is 2 bytes long
```suggestion
gov action ID - `11111111111111111111111111111111111111111111111111111111111111110000`
```

Reply:
there is no fixed length limit for index bytes, it just needs to be a valid hex at the end of a tx ID (fixed) bytes